ui/hunger_meter.gd

```diff
@@ -0,0 +1,7 @@
+class_name Hunger_Bar extends ProgressBar
+
+signal hunger_meter_empty
+
+func _on_value_changed(value: float) -> void:
+	if value == 0:
+		emit_signal("hunger meter empty")
```

Comment:
I don't think this works...? It's less error-prone to just do:
```gdscript
hunger_meter_empty.emit()
```
Because you will immediately get an error if the signal doesn't exist, as opposed to `emit_signal()` which I think goes looking for that signal by string at runtime, so you don't know it is broken until you run it.